### PR TITLE
[REVIEW] feat: implement expanded panels

### PR DESCRIPTION
**Pull request by mesteche**

### CODE REVIEW

File: src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  return (
+    <ExpansionPanel expanded={expanded} onChange={action('internal panel toggle called')} >
+      <Summary>
+        Summary
+        <Toggle expanded={expanded} onClick={toggle} />
+      </Summary>
+      <ExpansionPanelDetails>
+        Details
+      </ExpansionPanelDetails>
+      <ExpansionPanelActions>
```
Comment (monzey) [inline src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx]: Can't have actions inside Summary right ?

Comment (mesteche) [reply src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx]: Well, not as a feature of Summary or pannel, but nothing prevents us from putting a ButtonGroup  as the last child of ExpansionPannelSummary when we use it.
See the second story.

File: src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <ExpansionPanelSummary>
+      Summary
+    </ExpansionPanelSummary>
+    <ExpansionPanelDetails>
+      Details
+    </ExpansionPanelDetails>
+    <ExpansionPanelActions>
+      <ButtonGroup>
+        <Button size="small">Cancel</Button>
+        <Button size="small" color="primary">Save</Button>
```
[the remark this answers is not in the document]
Comment (mesteche) [reply src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx]: Yep, no problem, it's just the story anyway.

File: src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx [hunk 3]
```diff
@@ -54,7 +56,7 @@ export const controledExpansionPanel = () => {
     <ExpansionPanel expanded={expanded} onChange={action('internal panel toggle called')} >
       <Summary>
         Summary
-        <Toggle expanded={expanded} onClick={toggle} />
+        <Toggle expanded={expanded || undefined} onClick={toggle} />
```
Comment (monzey) [inline src/Molecules/ExpansionPanel/ExpansionPanel.stories.jsx]: Why ?

**Pull request merged**